LICENSE

```diff
@@ -1,27 +1,29 @@
+BSD 3-Clause License
+
 Copyright (c) Django Software Foundation and individual contributors.
 All rights reserved.
 
-Redistribution and use in source and binary forms, with or without modification,
-are permitted provided that the following conditions are met:
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-    1. Redistributions of source code must retain the above copyright notice,
-       this list of conditions and the following disclaimer.
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
 
-    2. Redistributions in binary form must reproduce the above copyright
-       notice, this list of conditions and the following disclaimer in the
-       documentation and/or other materials provided with the distribution.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
 
-    3. Neither the name of Django nor the names of its contributors may be used
-       to endorse or promote products derived from this software without
-       specific prior written permission.
+* Neither the name of the copyright holder nor the names of its
```

Comment:
Is the line wrapping copied from some place and the existing line wrapping doesn't work? Here, for example, it seems that "contributors" would fit on this line based on the wrapping of other lines.

Reply:
I agree that it would fit. The licence file I copied in is the one GitHub provides for new projects.